android/app/src/main/java/org/fossasia/openevent/activities/SessionDetailActivity.java

```diff
@@ -57,11 +58,6 @@
 import io.reactivex.functions.Consumer;
 import timber.log.Timber;
 
-
-/**
- * User: MananWason
```

Comment:
Any specific reason to delete the default file template?
I think we can leave it be.